.gitlab-ci.yml

```diff
@@ -103,6 +104,14 @@ check-wasm:
         cargo check --verbose --no-default-features --target wasm32-unknown-unknown --manifest-path crates/${crate}/Cargo.toml;
       done
 
+check-miri:
+  stage:                           check
+  <<:                              *docker-env
+  <<:                              *test-refs
+  script:
+    - for crate in ${MIRI_CRATES}; do
+        cargo miri test -p ${crate};
+      done
```

Comment:
Why do the crates have to be specified one by one instead of doing: `cargo miri test`?

Reply:
Some of the crates in lang, related to ir and AST, run incredibly slow in Miri. Instead of marking them one by one, I chose only the crates with possible significant unsafe.

Reply:
Gotcha. When `MIRI_CRATES` is declared, can you add a comment explaining that?